lib/util/dio_exception_ext.dart

```diff
@@ -3,17 +3,34 @@ import 'package:dio/dio.dart';
 extension PostcardExepctionExt on DioException {
   String get errorMessage => response?.data ?? '';
 
+  int get statusCode => response?.statusCode ?? 0;
+
   bool get isPostcardAlreadyStamped =>
       errorMessage == PostcardExceptionType.alreadyStamped.errorMessage;
+
+  bool get isPostcardClaimEmptyLimited =>
+      statusCode == PostcardExceptionType.claimEmptyLimited.statusCode;
 }
 
 enum PostcardExceptionType {
-  alreadyStamped;
+  alreadyStamped,
+  claimEmptyLimited;
```

Comment:
should be renamed to `tooManyRequest`. Some other apis maybe return the same code in the future.